selfie/bintray.gradle

```diff
@@ -1,6 +1,6 @@
 apply plugin: 'com.jfrog.bintray'
 
-version = '0.0.11'
+version = '0.1'
```

Comment:
Same as above. As this is a breaking change, I feel like it's time we bump this to 1.0.0